example/example.goconvey

```diff
@@ -0,0 +1 @@
+ignore
```

Comment:
what is this?

Reply:
goconvey thinks those folders are packages then complains that they don't have any go code in them